README.md

```diff
@@ -467,6 +467,30 @@ stacks:
       template: my-stack.rb
 ```
 
+### Loading SparklePacks
+
+[SparklePacks](http://www.sparkleformation.io/docs/sparkle_formation/sparkle-packs.html) can be pre-loaded using compiler options. This requires the name of a rubygem to `require` followed by the name of the SparklePack, which is usually the same name as the Gem.
+
+```yaml
+stacks:
+  us-east-1
+    my-stack:
+      template: my-stack-with-dynamic.rb
+      compiler_options:
+        sparkle_packs:
+          vpc-sparkle-pack: vpc-sparkle-pack
```

Comment:
You might need to change `sparkle_packs` to a list of sparkle packs to preserve order. I'm not sure if this structure will guarantee it. Ordering looks important if you want to override things. See [here](http://www.sparkleformation.io/docs/sparkle_formation/sparkle-packs.html#usage)